homeassistant/components/automation/__init__.py

```diff
@@ -240,6 +241,8 @@ async def reload_service_handler(service_call):
 class AutomationTrace:
     """Container for automation trace."""
 
+    _runids = count(0)
```

Comment:
We normally separate words with underscore in variable names (`_run_ids`).

Reply:
Fixed in #47669